sdk/java/src/main/java/com/chain/api/CoreConfig.java

```diff
@@ -0,0 +1,141 @@
+package com.chain.api;
+
+import com.chain.http.*;
+import com.chain.exception.*;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.*;
+
+public class CoreConfig {
+
+  public static class Info {
+    public String state;
+
+    @SerializedName("is_configured")
+    public Boolean isConfigured;
+
+    @SerializedName("configured_at")
+    public Date configuredAt;
+
+    @SerializedName("is_signer")
+    public Boolean isSigner;
```

Comment:
Generally, it looks like we use int and long in the SDK, not Integer and Long. I'd vote for consistency and use primitives across all objects.